Update error-accessing-app-from-teams-desktop.md (#435)
MicrosoftDocs/Viva: Viva/connections/troubleshoot/error-accessing-app-from-teams-desktop.md

## Resolution

To resolve this issue, navigate to the API Access page from the SharePoint admin center:

> 1. As a Global Admin, open the **SharePoint admin center**.

Reviewer: ```suggestion
1. As a Global admin, open the **SharePoint admin center**.
```

2. Select **Advanced**. 
3. Select **API access**.

> **Important** If you don't see the API access option under Advanced, try the following steps: 

Reviewer: ```suggestion
If you don't see the **API access** option under **Advanced**, follow these steps: 
```

> 1. As a Global Admin, open the **SharePoint admin center**.

Reviewer: ```suggestion
1. As a Global admin, open the **SharePoint admin center**.
```
